eclass/distutils-r1.eclass

```diff
@@ -178,7 +178,7 @@ esac
 if [[ ! ${_DISTUTILS_R1} ]]; then
 
 [[ ${EAPI} == 6 ]] && inherit eutils xdg-utils
-inherit multibuild multiprocessing toolchain-funcs
+inherit multibuild multiprocessing ninja-utils toolchain-funcs
```

Comment:
I wanted to ask what is the overhead of inheriting so much? Can we do the inherit when needed? (behind `[[ ${DISTUTILS_USE_PEP517} == "meson-python" ]]` for example)?

It is just that as an eclass with ~1k ebuilds, it might affect performance?

Reply:
It's a small eclass with no side effects and sourcing is normally done only when building metadata cache, so I don't think it's worth the added complexity. Feel free to do some benchmarks, though.

Reply:
My weak preference is to conditionally inherit it because it makes it a bit easier to understand how it's used.

Reply:
Should we also make `multiprocessing` inherit conditional to setuptools || maturin?

Reply:
Okay, you turned me against it 😛